src/vxworks/mod.rs

```diff
@@ -2167,6 +2018,12 @@ extern {
         iov: *const ::iovec,
         iovcnt: ::c_int,
     ) -> ::ssize_t;
+
+    // randomNumGen.h
+    pub fn randBytes(buf: *mut c_uchar, length: c_int) -> c_int;
+    pub fn randABytes(buf: *mut c_uchar, length: c_int) -> c_int;
+    pub fn randUBytes(buf: *mut c_uchar, length: c_int) -> c_int;
+    pub fn randSecure() -> BOOL;    
```

Comment:
@n-salim 
Should I change BOOL to c_int ? althoug the BOOL is defined as c_int here.

Reply:
Yes make it c_int to align to the proposed future signature.

Reply:
Sure.